tests/foreman/conftest.py

```diff
@@ -51,7 +51,7 @@ def pytest_report_header(config):
         now = datetime.datetime.utcnow()
         xml = config._store.get(xml_key, None)
         if xml:
-            xml.add_global_property('start_time', now)
+            xml.add_global_property('start_time', now.strftime("%Y-%m-%dT%H:%M:%S"))
```

Comment:
@rplevka Since we're hitting this inconsistency, please define a module constant like `FMT_XUNIT_TIME` for this format string, and use it both here and at L150 and L156 where the workers will be calling the fixtures to add these timestamps.

Reply:
thanks for the suggestion, I addressed your comment